symplyphysics/laws/waves/position_of_antinodes_in_standing_wave.py

```diff
@@ -0,0 +1,70 @@
+from sympy import Eq, S
+from symplyphysics import (
+    units,
+    dimensionless,
+    Symbol,
+    Quantity,
+    print_expression,
+    validate_input,
+    validate_output,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.definitions import angular_wavenumber_is_inverse_wavelength as wavenumber_def
+from symplyphysics.laws.waves import displacement_in_standing_wave as standing_wave_law
+
+# Description
+## In a standing wave, the locations of maximum amplitude are called antinodes. These locations are
+## not arbitrary, however, and are integer multiples of half the wavelength shifted by a quarter
+## of the wavelength:
+
+# Law: x_antinode = (m + 1/2) * (lambda / 2)
+## x_antinode - position of m-th antinode
+## m - integer
+## lambda - wavelength
+
+antinode_position = Symbol("antinode_position", units.length, real=True)
+integer_factor = Symbol("integer_factor", dimensionless, integer=True)
+wavelength = Symbol("wavelength", units.length, positive=True)
+
+law = Eq(antinode_position, (integer_factor + S(1)/2) * wavelength / 2)
```

Comment:
`1/2` does not work here?

Reply:
`1/2` would yield a `float` which is approximate but `S(1)/2` is exact. I think we have to keep fractions exact in formulas

Reply:
Ok. Maybe `S.One` looks a bit better.

Reply:
I agree